aio/content/examples/dynamic-component-loader/src/app/app.module.ts

```diff
@@ -9,16 +9,15 @@ import { AdService } from './ad.service';
 
 @NgModule({
   imports: [ BrowserModule ],
-  providers: [AdService],
-  declarations: [ AppComponent,
-                  AdBannerComponent,
-                  HeroJobAdComponent,
-                  HeroProfileComponent,
-                  AdDirective ],
+  providers: [ AdService ],
+  declarations: [
+    AppComponent,
+    AdBannerComponent,
+    HeroJobAdComponent,
+    HeroProfileComponent,
+    AdDirective ],
```

Comment:
Since you're at it already:

```suggestion
    AdDirective
  ],
```